features/steps/thens/alignment.py

```diff
@@ -110,3 +127,43 @@ def step_impl(context, inst, ifc_rep_criteria, ifc_layout_criteria):
                                 yield ValidationOutcome(inst=inst, expected=ifc_layout_criteria,
                                                         observed="IfcAlignmentCant",
                                                         severity=OutcomeSeverity.ERROR)
+
+
+@gherkin_ifc.step(
+    'The alignment\'s {layout_type} layout must have the same number of segments as the shape representation')
```

Comment:
```suggestion
@gherkin_ifc.step('The layout must have the same number of segments as the shape representation')
```

Perhaps we can write this even simpler

We have the layout type also implicitly as information, i.e. `inst.is_a()`

Reply:
Agree!  I implemented this per your suggested change.